### PR TITLE
_FFObjGet() - Function Headers

### PR DESCRIPTION
**Pull request by mlipok**
Just a some documentation fixes (Function Headers)

### CODE REVIEW

**Comment by Danp2** [comment]
Next-release branch has the latest code. Please edit PR and select that branch as base.

**Comment by mlipok** [comment]
Hope this is what you asked for ?

File: FF.au3 [hunk 1]
```diff
@@ -3447,12 +3447,12 @@ EndFunc   ;==>_FFXPath
...
-;                               | ID
-;                               | Name + index
-;                               | Class + index
-;                               | Tag-name + index
+; Parameter(s): .: $sElement    - string that represents html tag atribute name, or tag name which will be searched
+;                  $sMode       - Optional:
+;                               | "id" (Default) search for element with attribute "id" = $sElement
+;                               | "name" search for element with attribute "name" = $sElement and index = $iIndex
+;                               | "class" search for element with attribute "class" name = $sElement and index = $iIndex
+;                               | "tag"  search for element with "tag" name = $sElement and index = $iIndex
```
Comment (Danp2) [inline FF.au3:3455]: Need to add QuerySAll. ;-)

Comment (mlipok) [reply FF.au3:3455]: I'm trying to do this, but still learning how to use GitHub features.

Comment (mlipok) [reply FF.au3:3455]: as there is used 
StringLower($sMode)
I propose to change it to 

Case "queryall"
but I do not see:
`		
```
		Case "QuerySAll"

			$sRet = ".querySelectorAll('" & $sElement & "')[" & $iIndex & "]"
```
`
When I edit this update.

Comment (Danp2) [inline FF.au3]: For consistency with _FFClick, should probably use "selector"

Comment (mlipok) [reply FF.au3:3455]: ok, "selector" should be fine

**Comment by mlipok** [comment]
sure but "tag" is not attribute so this should be:
```
; Parameter(s): .: $sElement    - string that identifies the desired element
;                  $sMode       - specifies the mode of selection (Optional)
;                               | "id" (Default) - search for element using "id" attribute
;                               | "name"         - search for element using "name" attribute 
;                               | "class"        - search for element using "class" attribute
;                               | "tag"          - search for element using "tag" **name**

```

**Comment by Danp2** [comment]
Yeah... you're correct. Maybe simplify even further and just do it like this --
```
; Parameter(s): .: $sElement    - string that identifies the desired element
;                  $sMode       - specifies the mode of selection (Optional)
;                               | "id" (Default) - search for element by ID
;                               | "name"         - search for element by name
;                               | "class"        - search for element by class name
;                               | "tag"          - search for element by tag name

**Comment by mlipok** [comment]
ok.

**Comment by mlipok** [comment]
I'm not sure how to properly update this with querySelectorAll ( this was my mistake with starting request from bad branch).
I close this request and open new one.

**Pull request closed without merging**